### ISSUE TITLE
Experimental Design to replace Wizard Pattern

### ISSUE THREAD

**Issue by thehastingsp** [department-of-veterans-affairs/vets-design-system-documentation#1186]
## What
> Wizard pattern replacement. 

## Purpose
> The wizard pattern presented to users to decide what type of claim they are filing violates accessibility best practices, so proposing a one question per page pattern to replace it.

## Usage
> When a user is making a decision that will send them down a specific path or flow based on that decision. This scenario we are solving for is when users are filing a decision review: Supplemental Claim and deciding what type of Supplemental Claim they are filing. 

## Behavior
> The user needs to select a type of claim: Disability, Education, etc. which then sends them to a flow specific to that choice. The Wizard presented dynamic content that appeared based on a radio button choice, and I am proposing a continue button that presents a new page based on the choice. This solves for accessibility, especially screen readers. 

## Examples
> [Staging example of HLR wizard](https://staging.va.gov/decision-reviews/higher-level-review/request-higher-level-review-form-20-0996/start) form choice. 
> [Staging example of proposed pattern](https://staging.va.gov/decision-reviews/supplemental-claim/request-supplemental-claim-form-20-0995/start)

## Accessibility
> This is intended to solve for accessibility and provide more clarity of choice for screen readers. 

## Guidance
> Using one question per page vs. dynamic content changes simplifies the decision process, lessens cognitive load, and offers step-by-step guidance for users who may struggle with interactions.  

## Research (optional)
> We are planning to test this pattern in an [upcoming usability test](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/decision-reviews/Supplemental-Claims/Research/MVP%20Usability%20Research/researchplan.md). 

## Code (optional)
> @Mottie can you fill this in please. 

## Next steps
Work has been reviewed with the design council and approved for testing. I plan to bring research findings to the Design System Council.

Attaching proposed examples files here. 
![Benefit-Choice-Proposed](https://user-images.githubusercontent.com/68615337/188915393-ca07ee33-68db-4843-b8f7-fec9250722d6.png)

https://user-images.githubusercontent.com/68615337/188915420-f4b46fa9-c79f-407e-aa27-94d39c088ab9.mov

**Comment by andaleliz** [comment]
@thehastingsp this is super exciting! Have you talked to @AngelaFowler82 about this pattern? The feedback I've received from her is that a one question per page pattern is actually very annoying for screenreader users, at least with her level of experience.

If you can nudge your team at all to code the prototype so you can test with screenreader users in your usability study (vs on production as outlined in your plan), it would be worthwhile to evaluate how one thing per page affects usability for screenreader users with different levels of experience. Our team is considering this pattern too and would love to learn from your research.

**Comment by thehastingsp** [comment]
thanks @andaleliz I've been working closely with @joshkimux and will attend @AngelaFowler82 office hours to learn more. we may have some trade-offs as screen reader testing is limited in prototyping tools, but I'll talk with engineering about what we could try to do.

**Comment by caw310** [comment]
@thehastingsp this request is on the DSC agenda for the meeting on Oct 28.

**Comment by caw310** [comment]
Follow up from DSC meeting, the DST will add this to our backlog to add your example to the pattern in the DS.

**Comment by thehastingsp** [comment]
thanks @caw310 are there any specific actions I need to take?

**Comment by caw310** [comment]
Not at this time. When we work this, Natalie or Matt may reach out to you. Thanks!

**Comment by humancompanion-usds** [comment]
@thehastingsp - Can you point me to your Sketch file, if that is in fact the best place to pull examples from? Thanks.

**Comment by Mottie** [comment]
Supplemental Claims URL was change to: https://staging.va.gov/decision-reviews/supplemental-claim/file-supplemental-claim-form-20-0995 (the "request-supplemental" was changed to "file-supplemental"), and for some reason if the `/introduction` or `/start` are included in the url, the page shows as not found

**Comment by thehastingsp** [comment]
@humancompanion-usds thank you for your patience while I was changing teams - [the sketch file link is here](https://www.sketch.com/s/d2416db4-9a4f-4919-abe4-20ba4bdcfd89/v/eO73v2/p/3741C13E-E884-4D6C-8466-86BB04726ED6/canvas?posX=-4042.8251953125&posY=-75.80126953125&zoom=0.5)

**Comment by thehastingsp** [comment]
but the most accurate version will be the staging link Robin shared.
